### ISSUE TITLE
Eslint 9/ flat config support

### ISSUE THREAD

**Issue by se-panfilov** [nirtamir2/eslint-plugin-sort-export-all#18]
Hello,

First of all, thanks for the great plugin.

By any chance, are you planning to update the plugin to be compatible with EsLint 9/flat config?

Thanks in advance

**Comment by nirtamir2** [comment]
Yes - support for ESLint 9 and flat config. Is supposed. In the meantime - you can use the compatible eslint function around it and use ESLint v9
